### ISSUE TITLE
[Bug] Error when opening Mythic Dungeons Tool

### ISSUE THREAD

**Issue by KurtzPT** [fang2hou/ElvUI_WindTools#218]
Using 2.40 WindTools, when opening MDT I get the following error:

```
Message: ...Ons\ElvUI_WindTools\Modules\Skins\Widgets\Button.lua:73: Usage: SecureHookScript(object, method, [handler]): 'handler' - Handler specified does not exist at self[handler]
Time: Wed Jun  1 17:33:49 2022
Count: 1
Stack: ...Ons\ElvUI_WindTools\Modules\Skins\Widgets\Button.lua:73: Usage: SecureHookScript(object, method, [handler]): 'handler' - Handler specified does not exist at self[handler]
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "@Interface\AddOns\ElvUI\Libraries\Core\Ace\AceHook-3.0\AceHook-3.0.lua"]:139: in function <...ElvUI\Libraries\Core\Ace\AceHook-3.0\AceHook-3.0.lua:118>
[string "@Interface\AddOns\ElvUI\Libraries\Core\Ace\AceHook-3.0\AceHook-3.0.lua"]:411: in function `SecureHookScript'
[string "@Interface\AddOns\ElvUI_WindTools\Modules\Skins\Widgets\Button.lua"]:73: in function <...Ons\ElvUI_WindTools\Modules\Skins\Widgets\Button.lua:67>
[string "=(tail call)"]: ?
[string "=[C]"]: in function `SetScript'
[string "@Interface\AddOns\MythicDungeonTools\MythicDungeonTools.lua"]:974: in function `MakeSidePanel'
[string "@Interface\AddOns\MythicDungeonTools\MythicDungeonTools.lua"]:4589: in function <...ace\AddOns\MythicDungeonTools\MythicDungeonTools.lua:4537>
[string "@Interface\AddOns\MythicDungeonTools\MythicDungeonTools.lua"]:264: in function <...ace\AddOns\MythicDungeonTools\MythicDungeonTools.lua:263>
[string "=[C]"]: in function `ShowInterface'
[string "@Interface\AddOns\MythicDungeonTools\MythicDungeonTools.lua"]:32: in function `OnClick'
[string "@Interface\AddOns\Bazooka\Bazooka.lua"]:1418: in function <Interface\AddOns\Bazooka\Bazooka.lua:1415>

Locals: 
```

**Comment by dsheaffe** [comment]
You can just disable Skins->Widgets->Button for now to avoid the errors.

**Comment by Skullcan** [comment]
Yeah, that's what I did, I'm just reporting so ppl are aware.

**Comment by fang2hou** [comment]
fixed in 3f19c00ff07b680f95aae85dda6c238c8efa9399
